core/chainio/avs_subscriber.go

```diff
@@ -165,12 +164,14 @@ func (s *AvsSubscriber) SubscribeToNewTasksV3(newTaskCreatedChan chan *servicema
 			case newBatch := <-internalChannel:
 				s.processNewBatchV3(newBatch, batchesSet, newBatchMutex, newTaskCreatedChan)
 			case <-pollLatestBatchTicker.C:
-				latestBatch, err := s.getLatestTaskFromEthereumV3()
+				latestBatch, err := s.getLatestNotRespondedTaskFromEthereumV3()
```

Comment:
Maybe "Pending" is better than "NotResponded".

Reply:
+1